Vision/PID/README.md

```diff
@@ -0,0 +1,129 @@
+
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-search
+- azure-cognitive-services
+name: Analyze form sample skill for cognitive search
+urlFragment: azure-analyzeform-sample
+description: This custom skill extracts specific fields from the results of a trained form recognition.
+azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Vision/AnalyzeForm/azuredeploy.json
+---
+
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+- azure-search
+- azure-container-registry
+- azure-functions
+name: PID 
+urlFragment: azure-pid-drawing-sample
+description: This custom skill extracts specific product/equipment names from PI&D drawings.
+Deploy: Manually [deploy the container image as a Azure function](#deployment)
+---
+
+# PID Skill
+
+P&ID is a graphical representation of the actual process plant using various symbols that represent actual equipment. This skill is designed to extract equipment infomration  from specific instrument symbols in engineering diagrams. The skill uses the X, Y coordinates of text extracted by OCR to generate groupings of text based on proximity, vertical and horizontal seperation and alignment. 

```

Comment:
You might want to explain what P&ID means, and put a link to https://en.wikipedia.org/wiki/Piping_and_instrumentation_diagram

Reply:
```suggestion
P&ID is a graphical representation of the actual process plant using various symbols that represent actual equipment. This skill is designed to extract equipment information  from specific instrument symbols in engineering diagrams. The skill uses the X, Y coordinates of text extracted by OCR to generate groupings of text based on proximity, vertical and horizontal separation and alignment. 

```